shinkai-bin/shinkai-node/src/network/v2_api/api_v2_commands_tools.rs

```diff
@@ -2434,6 +2434,80 @@ impl Node {
         let _ = res.send(result).await;
         Ok(())
     }
+
+    pub async fn v2_api_store_proxy(
+        db: Arc<SqliteManager>,
+        bearer: String,
+        tool_router_key: String,
+        res: Sender<Result<Value, APIError>>,
+    ) -> Result<(), NodeError> {
+        // Validate the bearer token
+        if Self::validate_bearer_token(&bearer, db.clone(), &res).await.is_err() {
+            return Ok(());
+        }
+
+        let store_url = env::var("SHINKAI_STORE_URL")
+            .unwrap_or("https://shinkai-store-302883622007.us-central1.run.app".to_string());
```

Comment:
@guillevalin @acedward i think i saw another url more friendly somewhere in slack

Reply:
maybe a new PR? i will mnerge this one